R/stat-density.r

```diff
@@ -134,3 +148,26 @@ compute_density <- function(x, w, from, to, bw = "nrd0", adjust = 1,
     n = nx
   ), n = length(dens$x))
 }
+
+reflect_density <- function(dens, bounds, from, to) {
```

Comment:
Please add some comments explaining what the function does and what the parameters are.

Reply:
Done.